Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml

```diff
@@ -13,14 +14,90 @@
+                    </DataTrigger>
+                    <DataTrigger Binding="{Binding DeltaStatusIcon, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}" Value="Unchanged">
+                        <Setter Property="DataGridCell.Foreground" TargetName="xDeltaStatusIconImageMakerControl" Value="Green"/>
+                        <Setter Property="ToolTip" TargetName="xDeltaStatusIconImageMakerControl" Value="Un-Changed API" />
+                    </DataTrigger>
+                    <DataTrigger Binding="{Binding DeltaStatusIcon, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}" Value="Changed">
+                        <Setter Property="DataGridCell.Foreground" TargetName="xDeltaStatusIconImageMakerControl" Value="Orange"/>
+                        <Setter Property="ToolTip" TargetName="xDeltaStatusIconImageMakerControl" Value="Modified" />
+                    </DataTrigger>
+                    <DataTrigger Binding="{Binding DeltaStatusIcon, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}" Value="Deleted">
+                        <Setter Property="DataGridCell.Foreground" TargetName="xDeltaStatusIconImageMakerControl" Value="Red"/>
+                    </DataTrigger>
+                    <DataTrigger Binding="{Binding DeltaStatusIcon, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}" Value="Added">
+                        <Setter Property="DataGridCell.Foreground" TargetName="xDeltaStatusIconImageMakerControl" Value="Blue"/>
+                        <Setter Property="ToolTip" TargetName="xDeltaStatusIconImageMakerControl" Value="New API" />
+                    </DataTrigger>
+                    <DataTrigger Binding="{Binding DeltaStatusIcon, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}" Value="Avoided">
+                        <Setter Property="DataGridCell.Foreground" TargetName="xDeltaStatusIconImageMakerControl" Value="Purple"/>
+                    </DataTrigger>
+                </DataTemplate.Triggers>
+            </DataTemplate>
+            
         </Grid.Resources>
         
         <Grid.RowDefinitions>
+            <RowDefinition Height="50" x:Name="xCompareBtnRow" />
             <RowDefinition Height="300*"/>
         </Grid.RowDefinitions>
 
-        <ginger:ucGrid x:Name="xApisSelectionGrid"  Grid.Row="0" Margin="0,0,0,0" Visibility="Visible" Title="API's Source/s Content to Select From"  ShowRefresh="Collapsed" ShowEdit="Collapsed"  ShowAdd="Collapsed" ShowDelete="Collapsed" ShowClearAll="Collapsed" ShowUndo="Collapsed"  ShowUpDown="Collapsed"></ginger:ucGrid>
+        <DockPanel>
+            <UserControls:ucButton x:Name="xCompreExistingItemBtn" Grid.Row="0" VerticalAlignment="Center" HorizontalAlignment="Right"
+                      ButtonType="RoundTextAndImageButton" Click="BtnCompareAPIClicked" ButtonText="Compare APIs" ToolTip="Compare with Existing API Models"  ButtonFontImageSize="15" ButtonImageHeight="15"
```

Comment:
change ButtonText to "Compare with Exiting API Models"

Reply:
Not done. Button looks to big. Also, suggested by Users to keep it as "Compare APIs" only.